### ISSUE TITLE
Consistant capitalization in HTTP API

### ISSUE THREAD

**Issue by victorb** [filecoin-project/go-filecoin#1153]
Sometimes the HTTP API returns fields with capitalized keys and sometimes not. Maybe has something to do with streaming?

Example:

## Non-capitalized
```console
$ curl localhost:3453/api/orderbook/asks
{"price":"1","size":"5000000000","owner":"fcqup6fswj2w8kkkryyf85wtfz66j99tx85uvjgan","id":22}
{"price":"5","size":"10","owner":"fcqaany0rx436dwjyuy3u8hsel87z2wmgdvelea3z","id":5}
{"price":"2.5","size":"200000","owner":"fcqdsls5fz64x0xleupt6dnq9gtveke056q0zrsua","id":9}
{"price":"1","size":"3221225472","owner":"fcq3qu2y50rwaq5h9ewl25h7jcrxxl7wmd45sa8w3","id":15}
{"price":"3","size":"5000000","owner":"fcqdft745rauztq9adatanx6q3a4mv0rvtfl6gfuk","id":8}
{"price":"2.6","size":"200001","owner":"fcqyv0v8ctwzd9vd8gu4f683y6vyvyh8ru5ffyn3s","id":19}
{"price":"1","size":"10","owner":"fcq5y65n23xdkcx2ymakflxpxqhkvewnwswp0me52","id":0}
```

## Capitalized

```console
$ curl localhost:3453/api/wallet/addrs/ls
{"Address":"fcqw66dxelsajhpzz3t4z3mzmpueypedlqfef34w8"}

```

**Comment by mishmosh** [comment]
FYI @eingenito

**Comment by anorth** [comment]
Should be resolved by #3098
